libs/img/icons/icons.svg

```diff
@@ -115,4 +114,9 @@
      <path fill="currentcolor" d="M13,0H5A1,1,0,0,0,4,1V17a1,1,0,0,0,1,1h8a1,1,0,0,0,1-1V1A1,1,0,0,0,13,0ZM8.5,1h1a.5205.5205,0,0,1,.5.5.52.52,0,0,1-.5.5h-1A.5115.5115,0,0,1,8,1.5.512.512,0,0,1,8.5,1ZM9,17.55a1.05,1.05,0,1,1,1.05-1.05A1.05,1.05,0,0,1,9,17.55ZM13,15H5V3h8Z" />
   </symbol>
 
+  <symbol id="lock" viewBox="0 0 18 18">
+    <title>Lock</title>
```

Comment:
Is the title tag actually relevant or could it be removed?

Reply:
Good consideration. I think for this i'll remove it to keep things trim & since icons are being refactored (in progress). This stuff can be helpful for SEO in some usages (it could also be an inline attr. on the SVG element) which i'll keep in mind for this next iteration. Thanks